### ISSUE TITLE
missing/misplaced files

### ISSUE THREAD

**Issue by pixelzoom** [heldentodd/xray-diffraction#9]
Related to #1 (code review).

> - [ ] Are all required files and directories present?  ...

* mipmaps/ is missing license.json.  This is where you should give attribution for the protractor.png image.

* I'm also wondering if you needed to use mipmaps for the protractor image, because they are usually significantly larger than files in images/.   Ask me if you need more info about this.

* images/ is empty. Unless you convert mipmaps to images, you can delete images/.

* What is screenshot_XRD.png?  It's currently at the top-level of the repository.

* Consider adding a README.md, it's typical for all GitHub repos.

**Comment by pixelzoom** [comment]
@heldentodd FYI... Now that https://github.com/phetsims/chipper/issues/951 has been addressed, the missing mipmaps/license.json will result in a build failure.  E.g.:

```
% xray-diffraction/
% grunt
Running "lint-all" task

Running "report-media" task
>> not-annotated: xray-diffraction/mipmaps/protractor.png
Fatal error: There is an issue with the licenses for media types.
```

**Comment by heldentodd** [comment]
I copied the protractor mipmap over from Bending Light. I have now copied over the license file as well. screenshot_XRD.png is a screenshot of the program I used on my own prototype site as an icon for people to click on (https://tholden79.wixsite.com/mysite2/xrd-simulation). Technically it is not part of the proposed PhET. I'll move it somewhere else.

Images folder is deleted for now.

I added a README.md file. I borrowed extensively from the one for "Wave Interference".

I think that addresses all the issues here.
